src/IJ_mv/IJMatrix_parcsr.c

```diff
@@ -2190,7 +2205,7 @@ hypre_IJMatrixAssembleOffProcValsParCSR( hypre_IJMatrix       *matrix,
 
    num_recvs = send_proc_obj.length;
    argsort_contact_procs = hypre_CTAlloc(HYPRE_Int,  num_recvs, HYPRE_MEMORY_HOST);
-   for(i=0; i < num_recvs; i++)
+   for (i=0; i < num_recvs; i++)
```

Comment:
Should we have spaces before and after the equal sign? (`i = 0`)

Reply:
I'm not sure there is an option to enforce spaces before and after.  Let me know if you find one.  In general, I like to have the extra spaces, but it's not that critical I think.

Reply:
I found the option `--pad-oper / -p` at http://astyle.sourceforge.net/astyle.html for padding spaces around operators. Should we add it?

Reply:
I vote yes. Looking at their example
```
if (foo==2)
    a=bar((b-c)*a,d--);
becomes:

if (foo == 2)
    a = bar((b - c) * a, d--);
```
which is exactly what I liked it to be.

Maybe there are other operators we don't like to apply it. Anyone can give an example?